In [1]:
import numpy as np

In [12]:
A = np.array([[3.65**2, 0.25**2], [0.55**2, 6.05**2]]) #DeltaXY_V'sSqrd
print(A)
XYperVsqrd = np.dot( np.linalg.inv(A), np.array([[40.0**2], [45.0**2]]) ) #DeltaPositionsOnFab
print(XYperVsqrd)

[[13.3225  0.0625]
 [ 0.3025 36.6025]]
[[119.84268317]
 [ 54.33365449]]


In [13]:
XYperV = np.sqrt(XYperVsqrd)
print(XYperV)

[[10.9472683 ]
 [ 7.37113658]]


New XperV: 10.9472683 (a few percent different, but pretty similar)
New YperV: 7.37113658 (basically the same as before)

Old XperV: 10.1665
Old YperV: 7.3414